README.md

```diff
@@ -134,29 +134,75 @@ Sketches:
 --------------------------------------------------------------------------------
 <BASE64_ENCODED_DATA_POINTS|REFERENCE_TO_OTHER_DATA_FILE>
 ```
-
-Each Protocol has unique set of parameters. For example:
-- CPI
+Each Sync Protocol has an integer to identify it. The list of integers is as follows:
+1. CPISync
+2. CPISync_OneLessRound
+3. CPISync_HalfRound
+4. ProbCPISync
+5. InteractiveCPISync
+6. OneWayCPISync
+7. FullSync
+8. IBLTSync
+9. OneWayIBLTSync
+10. IBLTSetOfSets
+11. IBLTSync_Multiset
+12. CuckooSync
+13. BloomFilterSync
+14. MET_IBLTSync
+    
+Each Protocol has a unique set of parameters.
+ 
+CPI
 ```
 m_bar: <INT>
 bits: <INT>
 epsilon: <INT>
 partitions/pFactor(for InterCPISync): <INT>
 hashes: <true|false>
 ```
-- IBLT
+- m_bar: This is the maximum degree of the polynomial being used in the interpolation. It is analogous to the differences between the sets.
+- bits: Refers to the bit length used for encoding the coefficients of the polynomial.
+- epsilon: This is the tolerance parameter that represents an error margin that is acceptable in the sync.
+- partitions/pFactor: This determines the number of partitions to split the data into during the sync.
+- hashes: Indicates whether to use hashing during the synchronization.
+
+IBLT
 ```
 expected: <INT>
 eltSize: <INT>
 numElemChild: <INT>
 ```
-- Cuckoo
+- expected: Represents the expected number of elements to determine the IBLT size.
```

Comment:
the explanation is not clear - how does this expectation affect the IBLT and what are the consequences if it is too small/large?